src/Components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -547,47 +515,149 @@ const ResourcePicker: React.FC<IResourcePickerProps> = ({
+                            </div>
+                            {!resourcesState.isLoading &&
+                                resourcesState.adapterResult?.result?.data?.findIndex(
+                                    (r) => r.id === props.data.value.id
+                                ) === -1 && (
+                                    <IconButton
+                                        iconProps={{
+                                            iconName: 'Delete'
+                                        }}
+                                        styles={{
+                                            root: { padding: '0 0 0 12px' }
+                                        }}
+                                        onClick={(event) => {
+                                            event.stopPropagation();
+                                            handleOnRemove(props.data);
+                                        }}
+                                        title={t(
+                                            'resourcesPicker.removeFromList'
+                                        )}
+                                    />
+                                )}
+                        </>
+                    )}
+                </components.Option>
+            </div>
+        );
+    };
+
+    const formatCreateLabel = (inputValue: string) =>
+        `${t('resourcesPicker.useNonExistingResource')} "${inputValue}"`;
+    const noOptionsMessage = () => t('resourcesPicker.noOption');
+    const loadingMessage = () => loadingLabelText;
+    const customFilter = (option: any, inputValue: string) => {
+        if (!inputValue) return true;
+        if (
+            (isResourceOption(option.data) &&
+                option.data.label.includes(inputValue)) ||
+            option.data.__isNew__
+        ) {
+            return true;
+        }
+    };
+
     return (
         <div className={classNames.root}>
-            <VirtualizedComboBox
-                styles={classNames.subComponentStyles.comboBox()}
-                placeholder={placeholder()}
-                label={label}
-                options={options}
-                selectedKey={selectedKey}
-                allowFreeform={allowFreeform}
-                autoComplete={'on'}
-                required
-                disabled={disabled}
-                errorMessage={inputError}
-                text={selectedOption?.text} // to show the selectedOption text even there is input error
-                onChange={(_e, option, _idx, value) => {
-                    handleOnChange(option, value);
-                }}
-                onRenderLabel={handleOnRenderLabel}
-                onRenderOption={handleOnRender}
-            />
+            <Stack tokens={{ childrenGap: 4 }}>
+                {label && <Stack horizontal>{dropdownLabel}</Stack>}
+                <CreatableSelect
+                    aria-labelledby="resource-picker-dropdown-label"
```

Comment:
doesn't there need to be an element with this id? I think that hsould be the label on the line above right?

Reply:
yes, it is the id of that label